src/main/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder.java

```diff
@@ -540,7 +563,7 @@ private String getReportName(final Tool tool) {
     }
 
     private AnnotatedReport scanWithTool(final Run<?, ?> run, final FilePath workspace, final TaskListener listener,
-            final Tool tool) throws IOException, InterruptedException {
+                                         final Tool tool) throws IOException, InterruptedException {
```

Comment:
is this formatting correct? Why did you change this line, it seems unrelated?

Reply:
Yeah I added an argument there which was unnecessary.Removed it now.